pkg/registry/scripts/main.js

```diff
@@ -93,7 +93,7 @@
             /* Curtains related logic */
             function connect() {
                 $scope.curtains = { };
-                loader.watch("namespaces").then(function() {
+                discover().then(function() {
```

Comment:
This code was added because a project-admin cannot list namespaces, 
so to let project-admin access UI we check for discover() to pass